scss/_git_repo.scss

```diff
@@ -1,26 +1,73 @@
+// Variable colors
+$font-color: #676069;
+$border-color: #eee;
+$repo-with: 500px;
+$link-color: #0362d9;
+$button-color: aliceblue;
 .git-repo-areas {
   margin: auto;
 }
 
 .git-repo {
-  .repo-name {
-    font-size: 1.2rem;
+  padding: 25px 0 25px 0;
+  border-bottom: 1px solid $border-color;
+  max-width: $repo-with;
+  .repo-forked {
+    .repo-name {
+      font-size: 1.3rem;
+      text-decoration: none;
+      color: $link-color;
+      font-weight: 700;
+    }
+    span {
+      display: block;
+      color: #676069;
+      padding-top: 7px;
+      &.forked-name {
+        font-size: 0.8rem;
+      }
+      &.repo-description {
+        font-size: 0.9rem;
+      }
+    }
   }
-  .link-out {
-    position: relative;
-    padding: 0.3rem 0.8rem;
-    display: inline-block;
-    vertical-align: top;
-    height: 1rem;
-    &:before {
-      position: absolute;
-      content: "";
-      border-right: 0.5rem solid $red;
-      border-bottom: 0.5rem solid transparent;
-      width: 0.5rem;
-      height: 0.5rem;
-      display: inline-block;
-      vertical-align: top;
+  .repo-info {
+    font-size: 0.75rem;
+    padding-top: 15px;
+    display: table;
+    span {
+      display: table-cell; //With table-cell text and icon or color will align middle
```

Comment:
I've been using distable: table for wrapper and display: table-cell for childs cause I want to make it vertical align center.
Reference: https://css-tricks.com/centering-in-the-unknown/